app/models/application_record.rb

```diff
@@ -1,3 +1,42 @@
 class ApplicationRecord < ActiveRecord::Base
   self.abstract_class = true
+
+  def set_bad_method
+    not_used = true
```

Comment:
Bliss.AI: report

| Debt | Issue |
|---:|---|
| 0.5 | Useless assignment to variable - `not_used`. |